tests/unit/jobserver/test_forms.py

```diff
@@ -11,22 +11,90 @@
 def test_jobrequestcreateform_with_single_backend():
     backend = BackendFactory()
     choices = backends_to_choices([backend])
-    form = JobRequestCreateForm({"backend": backend.slug}, backends=choices)
+    form_data = {
+        "requested_actions": ["do_something"],
+        "backend": backend.slug,
+    }
+    form = JobRequestCreateForm(
+        ["do_something"],
+        backends=choices,
+        database_actions=[],
+        codelists_status="ok",
+        data=form_data,
+    )
 
     assert "backend" in form.fields
     assert form.fields["backend"].choices == choices
+    assert "requested_actions" in form.fields
+    assert form.fields["requested_actions"].choices == [
+        ("do_something", "do_something")
+    ]
 
-    assert form.is_valid, form.errors
+    assert form.is_valid(), (form.errors, form.non_field_errors())
 
 
 def test_jobrequestcreateform_with_multiple_backends():
+    for i in range(3):
+        BackendFactory()
```

Comment:
Factories all have a `create_batch` method that takes a number and the normal kwargs, so you can express this as:

    BackendFactory.create_batch(3)